src/plugins/data/public/search/aggs/buckets/geo_hash.ts

```diff
@@ -54,6 +54,7 @@ export const getGeoHashBucketAgg = ({ getInternalStartServices }: GeoHashBucketA
     {
       name: BUCKET_TYPES.GEOHASH_GRID,
       title: geohashGridTitle,
+      makeLabel: () => geohashGridTitle,
```

Comment:
Though not directly related to the other items in this PR, I added these `makeLabel` methods to geo hash & filter aggs in this PR because I noticed they were missing, which causes column headers to be unformatted in the inspector.